pkg/config/spec.go

```diff
@@ -852,14 +853,9 @@ func GetSourceSize(config *Config, source *v1.ImageSource) (int64, error) {
 
 		// This is always set for pods running under kubernetes
 		_, underKubernetes := os.LookupEnv("KUBERNETES_SERVICE_HOST")
-		config.Logger.Logger.Info().Bool("status", underKubernetes).Msg("Running under kubernetes")
 		// Try to get the HOST_DIR in case we are not using the default one
-		hostDir := os.Getenv("HOST_DIR")
-		// If we are under kubernetes but the HOST_DIR var is empty, default to /host as system-upgrade-controller mounts
-		// the host in that dir by default
-		if underKubernetes && hostDir == "" {
-			hostDir = "/host"
-		}
+		hostDir := k8sutils.GetHostDirForK8s()
+		config.Logger.Logger.Debug().Bool("status", underKubernetes).Str("hostdir", hostDir).Msg("Running under kubernetes host directory")
```

Comment:
There is no `if` anymore. This will always read "Running under kubernetes host directory" no? Or is `Bool()` a conditions on whether to print this or not?

Reply:
yes, indeed, I will change the msg.

this is just to show the values of the kubernetes and hostdirectory for debug purposes as it can happen that its doing it worngly and we have no visibility onto it